thorlcr/activities/hashdistrib/thhashdistribslave.cpp

```diff
@@ -610,6 +598,13 @@ class CDistributorBase : public CSimpleInterface, implements IHashDistributor, i
                     delete targets.item(n);
             }
         }
+        void reinit()
+        {
+            if (initialized)
+                reset();
+            else
+                init();
+        }
```

Comment:
Made public and called earlier (previously on thread start in CSender::process), because of the way an early stop is now signalled, could be before the CSender thread gets going.